qa/L0_model_queue/model_queue_test.py

```diff
@@ -115,12 +114,12 @@ def test_max_queue_size(self):
         # model execution. Then sends 10 requests to overload the model queue,
         # expecting 2 of the requests are returned with error code immediately.
         dtype = np.float32
-        shapes = ([16],)
+        shapes = ([16], )
 
         for trial in self.trials_:
             preceding_thread = threading.Thread(
                 target=self.check_response,
-                args=(8, dtype, shapes, 0, 0, (1999, 1000)),
```

Comment:
Why do you change the threshold values?

Reply:
This test isn't failing on master (without the change in threshold)

Reply:
Reverted these changes. Yes they were not needed.